src/commands/git-metadata/README.md

```diff
@@ -33,6 +33,8 @@ datadog-ci git-metadata upload
 
 #### Limitations
 
+This command will only report commits with a commit date within the last 30 days.
```

Comment:
Do we want to say `within the last 30 days, up to 1,000 commits.` ?

Reply:
yes we do 🙃